<launch>

    <node pkg="calib" name="preproc_cam3" type="preprocessing_node" respawn="true" output="screen" args="/cam3/qhd/points cam3" />
    <node pkg="calib" name="plane_detection_cam3" type="plane_detection_node" respawn="true" output="screen" args="cam3" />
    <node pkg="calib" name="plane_detection_scene" type="plane_detection_node" respawn="true" output="screen" args="scene" />
    <node pkg="calib" name="matching_cam3_scene" type="matching_node" respawn="true" output="screen" args="cam3 scene" />
    <node pkg="calib" name="transform_estimation_cam3_scene" type="transform_estimation_node" respawn="true" output="screen" args="cam3 scene" />
    <node pkg="calib" name="merging_cam3_scene" type="merging_node" respawn="true" output="screen" args="cam3 scene full" />
    <node pkg="calib" name="preproc_full" type="preprocessing_node" respawn="true" output="screen" args="/calib/clouds/full full" />
    <node pkg="calib" name="tf_filter_cam3_scene" type="tf_node" respawn="true" output="screen" args="cam_center cam3_scene" />
</launch>
